app/init.php

```diff
@@ -84,6 +84,7 @@
 const APP_PAGING_LIMIT = 12;
 const APP_LIMIT_COUNT = 5000;
 const APP_LIMIT_USERS = 10000;
+const APP_LIMIT_USER_SESSIONS = 100;
```

Comment:
I feel we should aim lower. Maybe 20.

Reply:
This is the maximum ever allowed? Are we sure to keep it low as 20?, default is 10 hardcoded

Reply:
this is also used the in the range validation of the endpoint `1-100` that's the max allowed and default. Do we keep separate constant for max allowed value and default? or make it lower as 10-20 ?